libqtile/backend/x11/window.py

```diff
@@ -427,7 +428,7 @@ def query_tree(self):
             parent = XWindow(self.conn, q.parent)
         return root, parent, [XWindow(self.conn, i) for i in q.children]
 
-    def paint_borders(self, colors, borderwidth, width, height):
+    def paint_borders(self, depth, colors, borderwidth, width, height):
```

Comment:
Perhaps instead of passing it, just get it from `self.depth`?

Reply:
I thought about this, but depth is a property for Window, not XWindow: `AttributeError: 'XWindow' object has no attribute 'depth'`. That's why the width and height is also passed right?

Reply:
Ah of course - ignore me :)